src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int STRING_INDEX = 0;
+    private static final int RESULT_INDEX = 1;
+    private static final String SUPPLY_FIELD = "supply";
+    private static final String BUY_FIELD = "buy";
+    private static final String RESULT_FIELD = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String readData = readReport(fromFileName);
+        String report = getReport(readData);
+        writeReport(toFileName, report);
+    }
+
+    private String readReport(String fromFileName) {
```

Comment:
This method reads FILE - not report.

```suggestion
    private String readFile(String fromFileName) {
```